src/main/java/app/persistence/AccountMapper.java

```diff
@@ -0,0 +1,83 @@
+package app.persistence;
+
+import app.exceptions.AccountCreationException;
+import app.exceptions.DatabaseException;
+import io.javalin.http.Context;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+public class AccountMapper {
+    public static ArrayList<String> getAllEmailsFromAccount(ConnectionPool connectionPool) throws DatabaseException {
+        ArrayList<String> emails = new ArrayList<>();
+
+        String sql = "select email from account";
+
+        try (Connection connection = connectionPool.getConnection();
+             PreparedStatement ps = connection.prepareStatement(sql)) {
+
+            ResultSet rs = ps.executeQuery();
+            while (rs.next()) {
+                emails.add(rs.getString("email"));
+            }
+            return emails;
+
+        } catch (SQLException e) {
+            throw new DatabaseException("Fejl i at hente emails", "Error in getAllEmailsFromAccount", e.getMessage());
+        }
+    }
+
+    public static int getIdFromAccountEmail(String email, ConnectionPool connectionPool) throws AccountCreationException {
+        int accountId = 0;
+        String sql = "SELECT account_id FROM account WHERE email = ?";
+
+        try (Connection connection = connectionPool.getConnection();
+             PreparedStatement ps = connection.prepareStatement(sql)) {
+
+            ps.setString(1, email);
+            ResultSet rs = ps.executeQuery();
+
+            if (rs.next()) {
+                accountId = rs.getInt("account_id");
+            }
+
+        } catch (SQLException e) {
+            throw new AccountCreationException("Fejl ved søgning efter account ID", "Error in getIdFromAccountEmail" ,e.getMessage());
```

Comment:
Tilføj den email der er blevet søgt efter til error messagen, f.eks.:
"Error in getIdFromAccountEmail for email: "+email